criteria.md

```diff
@@ -1,47 +1,84 @@
-| Build & Release | All released software assets MUST be created with consistent, automated build and release processes. |
-| Build & Release | All build and release pipelines MUST use standardized tooling to ingest dependencies at build time. |
-| Build & Release | All software releases MUST include a descriptive log of functional and security modifications. |
-| Documentation | The project documentation MUST include a policy for coordinated vulnerability reporting, with a clear timeframe for response. |
-| Documentation | The project documentation MUST include a mechanism for reporting defects. |
-| Documentation | The project documentation MUST include a guide for contributors and approvers that includes the requirements for acceptable contributions.|
-| Documentation | The project documentation MUST provide design documentation demonstrating all actions and actors within the system. |
-| Quality | All released software assets MUST be associated with a machine-readable list of all direct and transitive dependencies with their associated version identifier. |
-| Quality | Any automated commit status checks MUST pass or require manual intervention prior to merge. |
-| Quality | All codebases within the project MUST enforce security requirements as applicable to the status and intent of the codebase. |
-| Quality | The version control system MUST NOT contain generated executable artifacts. |
+| Access Control | The project's permissions in [CI/CD pipelines] MUST be configured to the lowest available privileges except when explicitly elevated. |
+| Access Control | The [project documentation] MUST have a policy that [code contributors] are reviewed prior to granting escalated permissions to sensitive resources. |
+| Build & Release | All [released software assets] MUST be created with consistent, automated [build and release pipelines]. |
+| Build & Release | All [build and release pipelines] MUST use standardized tooling to ingest dependencies at build time. |
+| Build & Release | All [releases] MUST include a descriptive log of functional and security modifications. |
+| Documentation | The [project documentation] MUST include a policy for coordinated [vulnerability reporting], with a clear timeframe for response. |
+| Documentation | The [project documentation] MUST include a mechanism for reporting [defects]. |
+| Documentation | The [project documentation] MUST include a guide for [code contributors] that includes the requirements for acceptable contributions. |
+| Documentation | The [project documentation] MUST provide design documentation demonstrating all actions and actors within the system. |
+| Quality | All [software assets] MUST be [released] with a machine-readable list of all direct and transitive dependencies with their associated version identifier. |
+| Quality | Any automated [status checks] for [commits]  MUST pass or require manual intervention prior to merge. |
```

Comment:
Projects may decide not to have integrated status checks for commits in the repo (especially if their CI is not integrated with the VCS - apart from pulling the codebase). Is it fine to say that in that context, this criteria may not be applicable?

Reply:
Yes! We will need to capture that in the detailed documentation